canvas/image.go

```diff
@@ -106,9 +203,7 @@ func NewImageFromFile(file string) *Image {
 // Since: 2.0
 func NewImageFromURI(uri fyne.URI) *Image {
 	if uri.Scheme() == "file" && len(uri.String()) > 7 {
-		return &Image{
-			File: uri.String()[7:],
-		}
+		return NewImageFromFile(uri.String()[7:])
```

Comment:
I know that this was wrong before, but you can use `uri.Path()` instead of slicing the string. Also, why does this call the function instead of how it was before given that the function doesn't do anything else?

Reply:
Good idea about uri.Path(). As for the why, I think it is better practice to reuse code rather than duplicate even for simple case.

Reply:
I see. I thought there was an inconsistency because the others were not calling the functions but they are using other parameters. Totally agree with you. Sorry for the fuzz.